test/src/test/testtool/BlockEnabledSenders/common.properties

```diff
@@ -9,7 +9,7 @@ jdbc.deleteTableIbisTemp.waitBeforeRead=1000
 jdbc.selectTableIbisTemp.className=nl.nn.adapterframework.jdbc.FixedQuerySender
 jdbc.selectTableIbisTemp.datasourceName=${jdbc.datasource.default}
 jdbc.selectTableIbisTemp.queryType = select
-jdbc.selectTableIbisTemp.readQuery=SELECT * FROM IBISTEMP WHERE TNUMBER=${jdbc.blockenabledsenders.key} ORDER BY TKEY
+jdbc.selectTableIbisTemp.readQuery=SELECT TKEY,TCHAR,TVARCHAR,TNUMBER,TTIMESTAMP,TDATE,TBLOB1,TCLOB,TBLOB2 FROM IBISTEMP WHERE TNUMBER=${jdbc.blockenabledsenders.key} ORDER BY TKEY
```

Comment:
Wouldn't it be possible to to modify some outcomes slightly, and keep this query simple?